### PR TITLE
Created a test suite with integration and unit tests

### LINKED ISSUE #42: Add tests setup in backend

### PR DESCRIPTION
**Pull request by ankurnarkhede**
Created a test suite with unit and integration tests

Closes #42

### CODE REVIEW

File: test/integration/health.test.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const chai = require("chai");
```
Comment (yesyash) [inline test/integration/health.test.ts:1]: Can we use ES modules instead of common js here, so that it it consistent across the repo?

**Comment by yesyash** [comment]
Can you please add `type: module` in package.json, this would also require changes to be made in tsconfig.
Here is a link to the TS docs that can help: [ECMAScript modules in Node.js](https://www.typescriptlang.org/docs/handbook/esm-node.html)

File: test/integration/health.test.ts (hunk 1 above)
Comment (ankurnarkhede) [reply test/integration/health.test.ts]: This is resolved.

**Comment by ankurnarkhede** [comment]
> type: module

Tried this change, getting the error
```
app.ts(3,28): error TS2792: Cannot find module './middlewares'. Did you mean to set the 'moduleResolution' option to 'node', or to add aliases to the 'paths' option?
```
which requires all module imports to be explicitly imported with `.js` extension. Any idea around this?

Or can we merge the changes in the current PR first if the type module changes are not a blocker?
@yesyash

**Comment by ankurnarkhede** [comment]
> Can you please add `type: module` in package.json, this would also require changes to be made in tsconfig. Here is a link to the TS docs that can help: [ECMAScript modules in Node.js](https://www.typescriptlang.org/docs/handbook/esm-node.html)

Adding an another PR for this.

**Pull request merged**